### PR TITLE
[SYTO-1099] Added a mixin for verlag and haas with the new rules

### PR DESCRIPTION
**Pull request by krlospelaez**
To test:

- Look for Verlag text and verify that it contains:
```css
letter-spacing: 0.2em;
text-transform: uppercase;
word-spacing: 0.05em;
```
- Look for Hass text and verify that it contains:
```css
letter-spacing: 0.04em;
```

### CODE REVIEW

File: src/styles/theme/_captions.scss [hunk 1]
```diff
@@ -1,37 +1,29 @@
...
+  @extend %mt3_haas;
   font-size: 0.875rem;
   font-weight: 400;
-  letter-spacing: 0.05em;
   line-height: 1.43;
 }
 
 .mt3_caption-title {
-  font-family: $mt3_haas-stack;
+  @extend %mt3_verlag;
```
Comment (arieldelfor) [inline src/styles/theme/_captions.scss]: you are applying text-transform: uppercase here. Are you sure you want to add uppercase to the mixin?

File: src/styles/themes/default/vars/_typography-fontfaces.scss [hunk 2]
```diff
@@ -39,3 +39,16 @@
 $mt3_haas-stack: 'Haas', 'Franklin Gothic Medium', 'Franklin Gothic', 'ITC Franklin Gothic', Arial, sans-serif;
 $mt3_verlag-stack: 'HCo Verlag', 'CenturyGothic', AppleGothic, Gill Sans, Gill Sans MT, sans-serif;
 $mt3_georgia-stack: Georgia, serif;
+
+
+%mt3_verlag {
+  font-family: $mt3_verlag-stack;
+  letter-spacing: 0.2em;
+  text-transform: uppercase;
```
Comment (arieldelfor) [inline src/styles/themes/default/vars/_typography-fontfaces.scss]: Not sure if it is convenient to add uppercase here.

Comment (krlospelaez) [inline src/styles/themes/default/vars/_typography-fontfaces.scss]: I know this is weird but it's part of the AC that every place using verlag must have uppercase.

File: src/modules/showcard/_styles.scss [hunk 3]
```diff
@@ -60,14 +60,12 @@
...
+    @extend %mt3_verlag;
     font-weight: 400;
-    letter-spacing: 0.1em;
     line-height: 1.05;
     /*horizontally center show card text*/
     text-align: left;
     vertical-align: middle;
-    word-spacing: .05em;
   }
   &-heading, &-title, &-airdate {
```
Comment (sebasrgl) [inline src/modules/showcard/_styles.scss]: is still this needed?

File: src/modules/showcard/_styles.scss [hunk 4]
```diff
@@ -60,14 +60,12 @@
...
     text-transform: uppercase;
```
Comment (krlospelaez) [inline src/modules/showcard/_styles.scss]: You're right, removing it

**Comment by krlospelaez** [comment]
PR updated

**Comment by sebasrgl** [comment]
PR OK! @arieldelfor you have something else to add?

**Pull request merged**